test/unit_test/rag/advanced_rag/knowlege_compile/conftest.py

```
@@ -78,15 +78,15 @@ def _message_fit_in(*args, **kwargs):
 # MatchDenseExpr at wiki_incremental module level).  If import fails,
 # create a minimal class-based stub instead of using MagicMock.
 try:
-    import common.doc_store.doc_store_base  # noqa: F401
+    import common.doc_store.doc_store_base
 except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)